javabot/Observable.java

```diff
@@ -0,0 +1,331 @@
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+
+/**
+ * A class to represent the state of a player
+ * */
+class PlayerInfo {
+  private int spent;
+  private int stack;
+  private boolean active;
+
+  /**
+   * @return the amount the player has spent in this game
+   */
+  public int getSpent(){
+    return spent;
+  }
+
+  /**
+   * @return the amount of money the player has left
+   */
+  public int getStack(){
+    return stack;
+  }
+
+  /**
+   * whether the player is still active in the game
+   * @return true if the player has not folded, false otherwise
+   */
+  public boolean getActive(){
+    return active;
+  }
+
+  public String toString(){
+    return "spent: " + spent + ", stack: " + stack + ", active: " + active;
+  }
+}
+
+/**
+ * A class to represent the action of a player
+ */
+class ActionInfo {
+
+  private int player;
+  private int action;
+
+  /**
+   * @return the index of the player taking this action
+   */
+  public int getPlayer(){
+    return player;
+  }
+
+  /**
+   * @return the action of the player
+   */
+  public int getAction(){
+    return action;
+  }
+
+  public String toString(){
+    String playerString = "Player " + player + ": ";
+    if(action == 0){
+      return(playerString+ "Fold");
```

Comment:
```suggestion
      return(playerString + "Fold");
```
Can I bet a space 👉👈